extensions/amp-lightbox/1.0/amp-lightbox.css

```diff
@@ -18,4 +18,18 @@ amp-lightbox {
   width: 100%;
   height: 100%;
   position: fixed;
+  visibility: hidden;
+  box-sizing: border-box;
```

Comment:
Let's also add `box-sizing: border-box` to the JSS file. It's necessary on Preact level as well.